src/components/types.ts

```diff
@@ -28,3 +28,43 @@ export type JbrowseFileInput = {
 };
 
 export type JbrowseFileFormatDict = Record<JbrowseFileInputFormats, string>;
+
+export type JbrowseAssemblyName = string;
+
+export type JbrowseCircularDefaultSession = {
```

Comment:
not able to get default session types out of jbrowse. I talked to caroline about it

Reply:
We can potentially assist them in setting that up, if they need a hand?